torchci/test/labelBot.test.ts

```diff
@@ -0,0 +1,159 @@
+    probot = utils.testProbot();
+    probot.load(labelBot);
+  });
+
+  afterEach(() => {
+    nock.cleanAll();
+    jest.restoreAllMocks();
+  });
+
+  test("random pr comment no reactoin", async () => {
+    const event = require("./fixtures/pull_request_comment.json");
+    const scope = nock("https://api.github.com");
+    await probot.receive(event);
+    if (!scope.isDone()) {
+      console.error("pending mocks: %j", scope.pendingMocks());
+    }
+    scope.done();
+  });
+
+  test("random issue comment no event", async () => {
+    const event = require("./fixtures/issue_comment.json");
+    const scope = nock("https://api.github.com");
+    await probot.receive(event);
+    if (!scope.isDone()) {
+      console.error("pending mocks: %j", scope.pendingMocks());
+    }
+    scope.done();
+  });
+
+  test("label comment on issue triggers confused reaction", async () => {
+    const event = require("./fixtures/issue_comment.json");
+    event.payload.comment.body = "@pytorchbot label enhancement";
+
+    const owner = event.payload.repository.owner.login;
+    const repo = event.payload.repository.name;
+    const comment_number = event.payload.comment.id;
+    const scope = nock("https://api.github.com")
+      .post(
+        `/repos/${owner}/${repo}/issues/comments/${comment_number}/reactions`,
+        (body) => {
+          expect(JSON.stringify(body)).toContain('{"content":"confused"}');
+          return true;
+        }
+      )
+      .reply(200, {});
+    await probot.receive(event);
+    if (!scope.isDone()) {
+      console.error("pending mocks: %j", scope.pendingMocks());
+    }
+    scope.done();
+  });
+
+  test("label comment with one label on pull request triggers add label and like", async () => {
+    const event = require("./fixtures/pull_request_comment.json");
+
+    event.payload.comment.body = "@pytorchbot label enhancement";
+
+    const owner = event.payload.repository.owner.login;
+    const repo = event.payload.repository.name;
+    const pr_number = event.payload.issue.number;
+    const comment_number = event.payload.comment.id;
+    const scope = nock("https://api.github.com")
+      .get(`/repos/${owner}/${repo}/labels?per_page=100&page=0`)
+      .reply(200, existingRepoLabelsResponse)
+      .post(
+        `/repos/${owner}/${repo}/issues/comments/${comment_number}/reactions`,
+        (body) => {
+          expect(JSON.stringify(body)).toContain('{"content":"+1"}');
+          return true;
+        }
+      )
+      .reply(200, {})
+      .post(`/repos/${owner}/${repo}/issues/${pr_number}/labels`, (body) => {
+        expect(JSON.stringify(body)).toContain(`{"labels":["enhancement"]}`);
+        return true;
+      })
+      .reply(200, {});
+    await probot.receive(event);
+    if (!scope.isDone()) {
+      console.error("pending mocks: %j", scope.pendingMocks());
+    }
+    scope.done();
+  });
+
+  test("label comment with several labels on pull request triggers add label and like", async () => {
```

Comment:
Would be nice to add `label comment with new label results in comment`

Reply:
Not entirely sure what do you mean. Is it a suggestion for a new unit test or to edit the description for this test? If second, how do you see the full description then?

Reply:
Oh, I see you are already test that unknown labels trigger comment in this test on line 146-150